src/main/java/ysg/fintech/service/AccountService.java

```diff
@@ -20,21 +24,53 @@ public class AccountService {
     public AccountDto createAccount(AccountDto accountDto){
         // dto > entity 로 변환
         AccountEntity account = AccountEntity.fromDto(accountDto);
+        // 계좌 개설 가능 여부 검증
+        validateCreateAccount(account);
         // entity > dto 로 변환 후 리턴
         return AccountDto.fromEntity(accountRepository.save(account));
     }
 
     // 계좌 해지
-    public AccountDto dropAccount(AccountDto.DropAccountDto dropAccountDto){
-        AccountEntity account = accountRepository.findById(dropAccountDto.getAccountIdx())
-                .orElse(new AccountEntity());
+    public AccountDto dropAccount(AccountDto accountDto){
+        AccountEntity account = accountRepository.findById(accountDto.getAccountIdx())
+                .orElseThrow(()-> new FintechException(ErrorCode.NOT_FOUND_ACCOUNT));
+        // 계좌 해지 가능 여부 검증
+        validateDropAccount(account);
         // 계좌 해지 설정
-        account.dropAccount(dropAccountDto);
+        account.dropAccount(accountDto);
         // entity > dto 로 변환 후 리턴
         return AccountDto.fromEntity(accountRepository.save(account));
     }
 
+    // 계좌 조회
+    public List<AccountDto> readAccount(AccountDto accountDto){
+        return accountRepository.findByMemberIdx(accountDto.getMemberIdx()).stream()
+                .map(AccountDto::fromEntity)
+                .collect(Collectors.toList());
+    }
+    // 계좌 개설 가능 여부 검증
+    private void validateCreateAccount(AccountEntity account){
+        // 사용자가 존재하지않는 경우
+        if(accountRepository.findByMemberIdx(account.getMemberIdx()).isEmpty()){
```

Comment:
```suggestion
        if (accountRepository.findByMemberIdx(account.getMemberIdx()).isEmpty()) {
```
coding convention 을 잘 지킬 수 있도록 시도해보시면 좋겠네요 :) java 개발자들끼리 보통 style 을 맞추는데, 대표적으로 많이 맞추는 것이 아래와 같은 coding convention 입니다.

https://naver.github.io/hackday-conventions-java/

Reply:
말씀해주신 부분엔 어떤 코딩 컨벤션을 지키는게 좋은건가요??

Reply:
위 링크에 있는 모든 내용을 지키는게 좋습니다 ^^;